.github/workflows/pipeline.yaml

```diff
@@ -6,27 +6,114 @@ jobs:
+#         - {version: '1.15', image: 'kindest/node:v1.15.12@sha256:d9b939055c1e852fe3d86955ee24976cab46cba518abcb8b13ba70917e6547a6' }
+#    name: k8s ${{ matrix.configuration.version }}
+#    steps:
+#    - name: checkout
+#      uses: actions/checkout@v2
+#      with:
+#        fetch-depth: '5'
+#    - name: build local Image
+#      run: docker build -t metacontrollerio/metacontroller:dev .
+#    - name: Replace CRD mnifests to v1beta1
+#      run: |
+#        sed -i 's/crds-v1/crds-v1beta1/' manifests/production/kustomization.yaml
+#        for file in examples/**/*.yaml; do sed -i 's/apiextensions.k8s.io\/v1/apiextensions.k8s.io\/v1beta1/g' $file; done
+#    - name: setup kind
+#      uses: engineerd/setup-kind@v0.5.0
+#      with:
+#        version: "v0.9.0"
+#        image: ${{ matrix.configuration.image }}
+#    - name: Load metacontroller image
+#      run: |
+#        kind load docker-image metacontrollerio/metacontroller:dev
+#    - name: Apply metacontroller manifests
+#      run: |
+#        kubectl apply -k manifests/dev
+#        sleep 20s
+#        kubectl get pods -n metacontroller
+#    - name: Run e2e tests
+#      run: |
+#        cd examples
+#        ./test.sh catset
+#    - name: coverage-e2e
+#      uses: codecov/codecov-action@v1
+#      with:
+#        token: ${{ secrets.CODECOV_TOKEN }} # not required for public repos
+#        files: ./tmp/integration-coverage.out
+#        flags: e2e
+#        fail_ci_if_error: true # optional (default = false)
+  e2etests:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        configuration:
+         - {version: '1.16', image: 'kindest/node:v1.16.15@sha256:a89c771f7de234e6547d43695c7ab047809ffc71a0c3b65aa54eda051c45ed20' }
+         - {version: '1.17', image: 'kindest/node:v1.17.11@sha256:5240a7a2c34bf241afb54ac05669f8a46661912eab05705d660971eeb12f6555' }
+         - {version: '1.18', image: 'kindest/node:v1.18.8@sha256:f4bcc97a0ad6e7abaf3f643d890add7efe6ee4ab90baeb374b4f41a4c95567eb' }
+         - {version: '1.19', image: 'kindest/node:v1.19.1@sha256:98cf5288864662e37115e362b23e4369c8c4a408f99cbc06e58ac30ddc721600' }
+    name: k8s ${{ matrix.configuration.version }}
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: '5'
+    - name: build local Image
+      run: docker build -t metacontrollerio/metacontroller:dev .
+    - name: setup kind
+      uses: engineerd/setup-kind@v0.5.0
+      with:
+        version: "v0.9.0"
+        image: ${{ matrix.configuration.image }}
+    - name: Load metacontroller image
+      run: |
+        kind load docker-image metacontrollerio/metacontroller:dev
+    - name: Apply metacontroller manifests
+      run: |
+        kubectl apply -k manifests/dev
+        sleep 20s
+        kubectl get pods -n metacontroller
+    - name: Run e2e tests
+      run: |
+        cd examples
+        ./test.sh catset
```

Comment:
Is it possible to provide catset as part of arg.
I took sometime to understand that catset is actually excluded as part of this test

Reply:
yeah, it was quick hack ;) I can try to make it proper posarg